pages/knowledge/colours-and-contrast/user-interface-components/README.md

```diff
@@ -0,0 +1,52 @@
+---
+navigation_title: "Colour contrast for UI components"
+position: 4
+changed: "2019-04-21"
+---
+
+# Colour contrast for User Interface components
+
+**“User Interface (UI) components” are interactive elements, ranging from simple buttons and input fields to complex UI widgets. Just as for text there is a general minimal contrast level that needs to be reached to make their functionality and content accessible.**
+
+## Minimal contrast ratio
+
+Version 2.1 of the [Web Content Accessibility Guidelines (WCAG) specify a minimal contrast ratio](https://www.w3.org/TR/WCAG21/#non-text-contrast) of `3:1` against adjacent colour(s) for all User Interface (UI) components.
+
+## Example: form controls
+
+The following screenshot shows a very simple contact form
+
+![Low contrast form](_media/low-contrast-form.png)
+
+The borders of the input fields and the background colour of the “Send message” button have an insufficient contrast ratio of `1.5:1`. 
+
+Because of that, certain users may have a hard time finding the input fields and enter information. 
+
+They might also struggle to identify the button by its typical shape. However, the button label adds a lot of valuable information, making this case less urgent or even negligible: it has sufficient contrast, it offers a clear invitation to interact by using a verb (which is better than a generic “OK”) and its visual appearance (font, size) is different from the other text elements surrounding it.
```

Comment:
I'd love to see this added (for the button):

> the background color (that should indicate the user that the element is a button) also has a contrast ratio of only `1.5:1`

Reply:
Ok, add to text

Reply:
Done